config/tsconfig/cjs.json

```diff
@@ -0,0 +1,10 @@
+// /config/tsconfig/cjs.json
+// Typescript compiler configuration for building a CommonJS module
```

Comment:
I don't think comments are allowed in .json files - github is flagging them as red. 

I sometimes add a "description" field to the json object. Or we could change the file from .json to .ts and export the config.

Reply:
This is actually a `tsconfig.json` file, which allows comments - which is why the builds still work. However, GitHub does not realise that as it thinks (mostly correctly) that only files named `tsconfig.json` are TS compiler config files.

> I sometimes add a "description" field to the json object. Or we could change the file from .json to .ts and export the config.

The configuration for the TS compiler has to be a JSON file IIRC. If needed, I could remove the comments instead.

Reply:
Ok, then, yeah, I think we should 'nix the comments. Even if TypeScript allows it, there are other JSON parsers that would choke here.

Reply:
Ok, will do